torchvision/datasets/widerface.py

```diff
@@ -137,13 +137,13 @@ def parse_train_val_annotations_file(self) -> None:
                             {
                                 "img_path": img_path,
                                 "annotations": {
-                                    "bbox": labels_tensor[:, 0:4],  # x, y, width, height
-                                    "blur": labels_tensor[:, 4],
-                                    "expression": labels_tensor[:, 5],
-                                    "illumination": labels_tensor[:, 6],
-                                    "occlusion": labels_tensor[:, 7],
-                                    "pose": labels_tensor[:, 8],
-                                    "invalid": labels_tensor[:, 9],
+                                    "bbox": labels_tensor[:, 0:4].clone(),  # x, y, width, height
```

Comment:
Views on tensor cannot be pickled correctly. Meaning regardless of the v2 wrapper, `datasets.Widerface` has never worked in a spawn context.